app/assets/javascripts/code/services/google/_map.js

```diff
@@ -92,7 +95,8 @@ export const map = (
     },
 
     listen: function(name, callback, diffmap) {
-      return google.maps.event.addListener(diffmap || this.mapObj, name, _(callback).bind(this));
+      const cb = _(callback).bind(this);
```

Comment:
Just curious, is `bind(this)` necessary here?
Also `cb` is not the most creating name :D

Reply:
I'm not totally sure it's needed, I just left it there w/o investigating. It's possible that w/o that the `this` would be wrong when called in response to the event.